### DISCUSSION TITLE
Vaultwarden installed on synology NAS failed to launch with reporting an error: [Valutwarden service is not running.]

### DISCUSSION THREAD

**Discussion by zhanghengtian** [dani-garcia/vaultwarden#3225] [Q&A]
### Subject of the issue

Vaultwarden installed on synology NAS failed to launch with reporting an error: [Valutwarden service is not running.]
### Deployment environment

* vaultwarden version:
1.27.0

* Install method:
Synology SPK on DSM
* Clients used: 
Synology NAS DSM7.1
* Reverse proxy and version: 
N/A
* MySQL/MariaDB or PostgreSQL version: 
N/A
* Other relevant details:

### Expected behaviour

Expecting Vaultwarden SPK being installed and fired normaly.
### Actual behaviour

Install SPK failed with reported error.
after click the [OK] button the SPK button turns out to be [fix]. once click [fix]. still with that error.
### Troubleshooting data

<img width="657" alt="19d2afd74a2fcb35c480638c740d4de" src="https://user-images.githubusercontent.com/47552985/216818133-6b022fab-81f9-484e-b7ab-556b101da5fd.png">

**Comment by BlackDex** [comment]
I suggest to try and contact the maintainer of that specific app.
This isn't something officially maintained as far as i know. The logo is even incorrect.

**Comment by BobWs** [comment]
Why not just use the Docker version?

**Comment by geolap** [comment]
Hello,
Owning a DS920+, save you hassle : use portainer to manage docker images/stacks.
On any x86 Synology NAS assure you can connect to a root ssh session and follow the linux procedure below.
https://docs.portainer.io/start/install/server/docker/linux

Then, you connect to your portainer to create a new stack "vaultwarden". You go to the Editor tab and can past your docker compose in ti.
When you're ok with the conf, you save it or update it. It will deploy the stack.

If docker compose is something obscure to you, let's kick in with https://www.composerize.com/
It helped me a lot !

Here is my working docker_compose :
[docker_compose.txt](https://github.com/dani-garcia/vaultwarden/files/10675577/docker_compose.txt)
